src/main/java/com/redhat/developers/msa/hello/ZipkinResource.java

```diff
@@ -32,8 +32,12 @@
     @Produces
     @Singleton
     public Brave getBrave() {
+    	String zipkinUrl = System.getenv("ZIPKIN_SERVER_URL");
```

Comment:
Shouldn't it be http://zipkin-query:9411 ? 

Plus, It should be defined inside the Dockerfile since this service can't be used outside a container.